### PR TITLE
Promote from BLAIS5-2909 to main

### PR DESCRIPTION
**Pull request by social-surveys-blaise-concourse**
Auto generated by concourse:

Promote from BLAIS5-2909 to main

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,103 @@
...
+Run the "create_daybatches" Cloud Function:
+
+```
+python -c "from main import create_daybatches; create_daybatches(None, None)"
+```
+
+Run the "check_daybatches" Cloud Function:
+
+```
+python -c "from main import check_daybatches; check_daybatches(None, None)"
```
Comment (JAWilliamsONS) [inline README.md:90]: Should be `poetry run python -c "from main import check_daybatches; check_daybatches(None, None)"`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,103 @@
...
+BLAISE_API_URL="localhost:90"
+BLAISE_SERVER_PARK="gusty"
+NOTIFY_API_KEY="blah-blah-blah"
+TO_NOTIFY_EMAIL="blah@blah.blah"
+```
+
+Run the "create_daybatches" Cloud Function:
+
+```
+python -c "from main import create_daybatches; create_daybatches(None, None)"
```
Comment (JAWilliamsONS) [inline README.md:84]: should be `poetry run python -c "from main import create_daybatches; create_daybatches(None, None)"`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,103 @@
...
+gcloud compute start-iap-tunnel restapi-1 80 --local-host-port=localhost:90 --zone europe-west2-a
+```
+
+Create an .env file in the root of the project and add the following environment variables:
+
+| Variable | Description | Example |
+| --- | --- | --- |
+| BLAISE_API_URL | The RESTful API URL the application will use to get installed questionnaire data. | localhost:90 |
+| BLAISE_SERVER_PARK | The name of the Blaise server park. | gusty |
+| NOTIFY_API_KEY | The API key for the GOV.UK Notify service. | blah-blah-blah |
```
Comment (JAWilliamsONS) [inline README.md:69]: Maybe add a comment that if you are running the unit tests locally then 'NOTIFY_API_KEY' can literally be "blah-blah-blah"

File: main.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import os
+
+from dotenv import load_dotenv
+
+from functions.instrument_functions import get_installed_instrument_data, \
```
Comment (JAWilliamsONS) [inline main.py:5]: Could use - 

`from functions.instrument_functions import (get_installed_instrument_data, get_instruments_with_active_survey_day_today_and_cases ...) `

I think it looks better than backslash escapes `\` - up to you though :)

**Review by JAWilliamsONS — COMMENTED** [review]
Unit tests run locally - all passed 👍 
Visual inspection of the code looks sensible - made some comments on the `read.me` and the imports in `main.py`

File: README.md (hunk 1 above)
Comment (Riceyo) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (Riceyo) [reply README.md:69]: Done.

File: main.py (hunk 4 above)
Comment (Riceyo) [reply main.py]: Added a comment that local env vars aren't necessary for unit and behave tests.

File: README.md (hunk 2 above)
Comment (Riceyo) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (Riceyo) [reply README.md:69]: Added a comment that local env vars aren't necessary for unit and behave tests.

File: main.py (hunk 4 above)
Comment (Riceyo) [reply main.py]: I agree it looks better, done.

**Review by JAWilliamsONS — APPROVED** [review]
Unit tests pass
Run in my sandbox (jw02) - all fine
Code looks good 👍

**Pull request merged**